lua/oil/init.lua

```diff
@@ -686,7 +686,7 @@ local function maybe_hijack_directory_buffer(bufnr)
   end
   local replaced = util.rename_buffer(
     bufnr,
-    util.addslash(config.adapter_to_scheme.files .. vim.fn.fnamemodify(bufname, ":p"))
+    config.adapter_to_scheme.files .. vim.fn.fnamemodify(bufname, ":p")
```

Comment:
Fixes when Windows enters nvim via `nvim .` or a directory, it won't have a double slash. Ubuntu seems to work fine with this change

Reply:
Could you go into a bit more detail here? The `addslash` should only add a trailing `/` if none currently exists. Is the problem that on windows the buffer name ends with a `\` so it gets modified to `\/`?

Reply:
It was adding two slashes facing the same way.